source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,70 @@
+.. _java-rs-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. TODO: 
+.. Add this sentence after the last sentence in the paragraph for implementing reactive streams subscribers. 
+.. To learn more about the Java Reactive Streams API, see link here.
+
+.. procedure::
+   :style: connected
+
+    .. step:: Implement Reactive Streams Subscribers
+
+      Create a folder within the Java folder in your Maven or Gradle project named ``helpers``.
+      In the ``helpers`` folder, create a Java file named ``SubscriberHelpers``.
+      The Java Reactive Streams driver is an implementation of the Reactive
+      Streams API, which consists of three components: Publisher, Subscriber,
+      and Subscription. 
+      
+      The following code is an implementation of six
+      different ``Subscriber`` types used in your {+driver-short+} application.
```

Comment:
The query actually uses multiple subscribers, and I was only able to take out one, as a lot of them are extensions of each other (so I couldn't take out ConsumerSubscriber because I needed it for PrintDocumentSubscriber

Reply:
Requesting feedback from the tech reviewer: is there a way that we can condense and simplify the Subscribers for the purposes of the tutorial?